### ISSUE TITLE
Uncaught exception in LDDataManager

### ISSUE THREAD

**Issue by jkodumal** [launchdarkly/ios-client#27]
```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[NSURL initFileURLWithPath:]: nil string parameter'
```

Seen at LDDatamanager:104

**Comment by Automatt** [comment]
Opened https://github.com/launchdarkly/ios-client/pull/28

**Comment by cmavromoustakos** [comment]
Sorry for not responding yesterday. I tested with both versions of the bundle and it is working for me regardless of how I retrieve the bundle for the application. In other words I was not able to reproduce the issue.

Based on Apple's documentation on getting the contents of a bundle, it should not make a difference if we use mainBundle vs bundleForClass. They should both return the same result.

https://developer.apple.com/library/ios/documentation/CoreFoundation/Conceptual/CFBundles/AccessingaBundlesContents/AccessingaBundlesContents.html

I am curious to what exactly the root of this issue is and will look around to see if anything else pops out at me. Then again, this issue will be a non issue in the near future as we are migrating away from CoreData.

**Comment by drichelson** [comment]
Resolved in 1.0.0
